### DISCUSSION TITLE
Implicit flow implementation

### DISCUSSION THREAD

**Discussion by DaiNguyenTM** [soofstad/react-oauth2-pkce#160] [Q&A] [answered]
Hello Stig,
I've tried the component with the implicit flow (which does not include the code/token exchange phase), and it doesn't work.
Could you please let me know how to make it work with this implicit flow?
Thank you so much for your help.
Best regards,
D.

**Comment by soofstad** [comment, marked as answer]
Hi,
This package does not support "implicit flow", only "Authorization Code flow with PKCE".
Implicit flow is the old, unsecure way of doing things. You should be using auth code with pkce. :slightly_smiling_face:

**Reply by DaiNguyenTM** [reply]
Thank you !
